TESTS/ticker/main.cpp

```diff
@@ -32,12 +32,12 @@ using namespace utest::v1;
 #define TIMESTAMP_MAX_DELTA_BITS(bits)  ((uint64_t)(0x7 << ((bits) - 4)))
 #define TIMESTAMP_MAX_DELTA TIMESTAMP_MAX_DELTA_BITS(32)
 
-struct ticker_interface_stub_t { 
+struct ticker_interface_stub_t {
```

Comment:
This file just has cosmetic changes right? If so this should be moved into a PR which is on master.

Reply:
Removed this file from this PR.